database/gormdb_test.go

```diff
@@ -751,6 +751,54 @@ func TestGormDBUpdateCourse(t *testing.T) {
 	}
 }
 
+func TestGormDBCourseUniqueContraint(t *testing.T) {
```

Comment:
Please move this test to a new file `gormdb_course_test.go`. (We would like to move more of the tests in `gormdb_test.go` to other files to make it easier to find, instead of one very long file...)

Reply:
I moved this test, along with all course related tests I found to a new file `gormdb_course_test.go`.